utils/spacewalk-utils.spec

```diff
@@ -20,74 +21,71 @@
 %{!?pylint_check: %global pylint_check 1}
 %endif
 
-%if 0%{?fedora} || 0%{?rhel}
```

Comment:
AFAIK, this is not used anymore.

Reply:
> AFAIK, this is not used anymore.

Maybe @Conan-Kudo is using this?

Reply:
With the current state of the RHEL/Fedora macros, I'd suggest cleaning things up and start adding them when real support for the Server on top of Fedora/CentOS/RHEL is planned.

Remember this is a server package, not client.

Reply:
Also please keep in mind (now that I re-reviewed the code), that I am removing this whole section for Fedora/RHEL and SUSE/openSUSE, as I can't find where the macro `%apache_group` is used later at the SPEC.

Reply:
I'm okay with this going away given it's completely unused anyway.